authorization-server/src/modules/app/env-vars-validation-schema.ts

```diff
@@ -14,6 +14,7 @@ export const envVarsValidationSchema = Joi.object({
   PORT: Joi.number().default(3000),
   BIND: Joi.string().ip().default('127.0.0.1'),
 
+  CHAIN_ID: Joi.number().positive().default(73799),
```

Comment:
Is it possible to constrain chain id to given set?

Reply:
@JGiter , yes, it is possible with Joi.